### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by denisshevchenko**

### CODE REVIEW

**Comment by denisshevchenko** [comment]
@4z3 Please check it out when you can.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
+# Ares
+
+Ares is an application server for the [Microgram](https://github.com/zalora/microgram). We can use it to control different applications (for example, start and stop them, or get logs information) via simple commands.
+
+Ares uses two concepts:
+
+1. Application. There're many different applications we can work with via Ares.
+2. Service. Ares supports only two services: [Angel](https://github.com/zalora/angel), daemon for the process monitoring/management, and [Nginx](https://github.com/zalora/microgram/tree/master/pkgs/nginx).
```
Comment (ip1981) [inline README.md:8]: Angel is a fork.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+  "wtfdbFile": "/var/ares/wtfdb.json"
+}
+```
+
+## API
+
+Ares listens UNIX-socket, not TCP. You can use `curl` to make requests, for example via such a script:
+
+```bash
+curl -fsS --unix-socket "$RUN_DIR"/warp.sock -X "$METHOD" http:"$URL" "$ADDITIONAL_ARGS" | jq .
```
Comment (ip1981) [inline README.md:43]: missing `//` after `http:`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+- `$ADDITIONAL_ARGS` - additional arguments for `curl`.
+
+### Start
+
+Starts `ares` as a daemon.
+
+Command:
+
+```bash
+$ nohup ares CONFIG 0<&- 1>ares.stdout.log 2>ares.stderr.log &
```
Comment (ip1981) [inline README.md:60]: Nobody would start daemons like this.

File: README.md (hunk 1 above)
Comment (denisshevchenko) [inline README.md:8]: So what?

File: README.md (hunk 3 above)
Comment (denisshevchenko) [inline README.md]: I took it from here: https://github.com/zalora/eris/blob/master/apps/ares/default.nix#L75

File: README.md (hunk 2 above)
Comment (denisshevchenko) [inline README.md:43]: https://github.com/zalora/eris/blob/master/apps/ares/default.nix#L55

**Pull request merged**